x-pack/plugins/infra/public/components/asset_details/tabs/profiling/profiling.tsx

```diff
@@ -50,6 +78,30 @@ export function Profiling() {
           <Functions />
         </>
       ),
+      append: (
+        <Popover iconSize="s" iconColor="subdued" icon="questionInCircle">
+          <EuiText size="xs">
+            <FormattedMessage
+              id="xpack.infra.profiling.functionsInfoPopoverBody"
+              defaultMessage="The TopN functions view shows the most frequently sampled functions, broken down by CPU time, annualized CO2, and annualized cost estimates. You can use this view to identify the most expensive line of code on your host. {learnMoreLink}"
```

Comment:
```suggestion
              defaultMessage="Identify the most expensive lines of code on your host by looking at the most frequently sampled functions, broken down by CPU time, annualized CO2, and annualized cost estimates. {learnMoreLink}"
```

Reply:
LGTM @mdbirnstiehl 

@iogbole - you OK with it?